### PR TITLE
Add minty servers to the list

### PR DESCRIPTION
**Pull request by Volas171**
image

this pull is the same as #3885 but the owner told me at the screenshot above this comment to pull request it since they couldn’t  fix the conflicts.

### CODE REVIEW

**Comment by WilloIzCitron** [comment]
OK your server it's online 24/7

**Review by WilloIzCitron — APPROVED** [review]

**Comment by Volas171** [comment]
can confirm , it is 24/7

File: servers_v6.json [hunk 1]
```diff
@@ -52,6 +52,13 @@
...
   },
   {
+     "name": "MindustryBR",
+     "address": ["mindustryptbr.ddns.net", "mindustryptbr.ddns.net:4444", "mindustryptbr.myddns.me:6666", "mindustryptbr.myddns.me:5555"]
+  },
+  {
+     "name": "SubZero",
+     "address": ["minty-server.ddns.net"]
+  },
     "name": "ALEX",
```
Comment (Anuken) [inline servers_v6.json:62]: This breaks the JSON.

File: servers_v6.json [hunk 2]
```diff
@@ -52,7 +52,14 @@
...
-    "name": "ALEX",
-    "address": ["alexmindustry.ddns.net:6568", "alexmindustry.ddns.net:6569"]
+     "name": "MindustryBR",
+     "address": ["mindustryptbr.ddns.net", "mindustryptbr.ddns.net:4444", "mindustryptbr.myddns.me:6666", "mindustryptbr.myddns.me:5555"]
+  },
+  {
+     "name": "SubZero",
+     "address": ["minty-server.ddns.net"]
+  },
+     "name": "ALEX",
```
Comment (Anuken) [inline servers_v6.json:62]: nope, still missing an opening brace

**Comment by Volas171** [comment]
ill just close and open again, idk why the fix isn't working

**Comment by DeltaNedas** [comment]
how do you mess up json

**Comment by Volas171** [comment]
idk , it wasn't even supposed to mess up

**Pull request closed without merging**